src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2242,9 +2242,8 @@ namespace HVACUnitarySystem {
 				if ( ! allocated( UnitarySystem( UnitarySysNum ).MSCoolingSpeedRatio ) ) UnitarySystem( UnitarySysNum ).MSCoolingSpeedRatio.allocate( UnitarySystem( UnitarySysNum ).NumOfSpeedCooling );
 			}
 
-			OnOffAirFlowRatio = 1.0;
-			PartLoadRatio = 1.0;
-			SetOnOffMassFlowRate( UnitarySysNum, OnOffAirFlowRatio, PartLoadRatio );
+			// set the multi-speed high flow rate variable in case a non-zero air flow rate resides on the coil inlet during sizing (e.g., upstream system ran prior to this one)
+			MSHPMassFlowRateHigh = 1.0; // doesn't matter what this value is since only coil size is needed and CompOn = 0 here
```

Comment:
@rraustad I also checked sizing values in eio files from both runs and find only difference is cooling capacity:

The existing program is unable to size the cooling capacity and issued a fatal message.

The fixed program is able to generate the cooling capacity shown below: 
 Component Sizing Information, AirLoopHVAC:UnitarySystem, ZONE RETAIL BACK_SPACE A  - STORY GROUND PSZ-VAV UNITARY COOLING, Design Size Nominal Cooling Capacity [W], 3081.61625.

Therefore, the value of MSHPMassFlowRateHigh does not matter. However, in order to pass a error check, the value has to be > 0.0. It seems that "MSHPMassFlowRateHigh = 1.0" looks too arbitrary. Can we set up as design flow rate?

Reply:
I could set it to EqSizing.CoolingAirVolFlow if that would look better. The coil is not simulated here since CompOp = 0 and jumps past the code in DXCoil to where the coil is off. But I don't disagree that using 1.0 is misleading even though the comment says it doesn't matter what this value is. Since you are on this branch can you make that change?